### PR TITLE
support ewr_num_images for ZTF TOOs

### PR DESCRIPTION
**Pull request by ebellm**
**Does this pull request make any changes to the database?**
No.

 Addresses #141

### CODE REVIEW

File: growth/too/templates/plan_manual.html [hunk 1]
```diff
@@ -67,6 +67,39 @@
...
+                    <label for=subprogram_name>Subprogram name</label>
+                    {{ render_field(form.subprogram_name, placeholder='Subprogram name') }}
+                <small id="balanceHelpBlock" class="form-text text-muted">
+                GW, GRB, Neutrino, NEO, etc.
+                </small>
+                </div>
+            </div>
+            <div class=form-row>
+                <div class="form-group col-sm-3">
+                    <button type=submit id=create class="btn btn-primary" name="btnform">Create</button>
```
Comment (lpsinger) [inline growth/too/templates/plan_manual.html:95]: What's the new submit button for?

File: growth/too/templates/plan_manual.html [hunk 2]
```diff
@@ -67,6 +67,39 @@
...
+            </div>
+            </div>
+            <div class=form-row>
+                <div class="form-group col-sm-6">
+                    <label for=plan_name>Queue name</label>
+                    {{ render_field(form.queue_name, placeholder='Queue name') }}
+                </div>
+                <div class="form-group col-sm-6">
+                    <label for=subprogram_name>Subprogram name</label>
+                    {{ render_field(form.subprogram_name, placeholder='Subprogram name') }}
```
Comment (lpsinger) [inline growth/too/templates/plan_manual.html:87]: Is freeform text really accepted here, or are there only specific options that are accepted?

File: growth/too/templates/plan_manual.html [hunk 3]
```diff
@@ -67,6 +67,39 @@
                     0 for standard imaging and readout
                     </small>
                 </div>
+                <div class="form-group col-sm-4">
+                    <label for=mode_num>Number of EWR Images (ZTF Only)</label>
```
Comment (lpsinger) [inline growth/too/templates/plan_manual.html:71]: Suggest adding a tooltip to explain what this option does.

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: growth/too/templates/plan_manual.html (hunk 1 above)
Comment (ebellm) [reply growth/too/templates/plan_manual.html]: Sorry, this was duplicated in error.

File: growth/too/templates/plan_manual.html (hunk 2 above)
Comment (ebellm) [reply growth/too/templates/plan_manual.html]: Yes, freeform text is accepted--this isn't actually new code though, I accidentally duplicated it.

File: growth/too/templates/plan_manual.html (hunk 3 above)
Comment (ebellm) [reply growth/too/templates/plan_manual.html:71]: Does the `<small>` text below work, or is there some other way that you implement that?

Comment (lpsinger) [reply growth/too/templates/plan_manual.html:71]: I'd like to see in the help text an explanation of what EWR stands for and what it means.

**Comment by lpsinger** [comment]
Please rebase. The unit tests are now passing on main.

**Review by lpsinger — APPROVED** [review]

**Pull request merged**